.dockerignore

```diff
@@ -0,0 +1,7 @@
+Dockerfile
```

Comment:
We should make sure to make this file as strict as possible, otherwise all this stuff gets included in `COPY . .` and busts the cache for all sorts of changes to various files.